packages/core/src/http-client/http-client.spec.ts

```diff
@@ -306,4 +306,41 @@ describe('generic http client', () => {
       ).rejects.toThrowErrorMatchingSnapshot();
     });
   });
+
+  describe('executeRawHttpRequest', () => {
+    beforeAll(() => {
+      nock.cleanAll();
+    });
+
+    it('takes a generic HTTP request and executes it', async () => {
+      nock('https://example.com', {
+        reqheaders: {
+          authorization: 'Basic VVNFUk5BTUU6UEFTU1dPUkQ=',
+          'sap-client': '001'
+        }
+      })
+        .get('/api/entity')
+        .query({
+          a: 'a',
+          b: 'b'
+        })
+        .reply(200, { res: 'ult' }, { sharp: 'header' });
+
+      const config = {
+        method: HttpMethod.GET,
+        url: '/api/entity',
+        params: {
+          a: 'a',
```

Comment:
The parameters are not asserted - what do they add to the test? Matching the mock implicitly tests they the parameters are considered, but this done by axios.

Reply:
I'll simplify the test.